code/modules/admin/verbs/randomverbs.dm

```diff
@@ -45,6 +45,33 @@
 	admin_ticket_log(M, msg)
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Subtle Message") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/cmd_admin_centcom_message(mob/M in GLOB.mob_list)

```

Comment:
Put the verb on /datum/admins instead

Reply:
but why and how?

Is that even possible to still have it as a right click and regular verb if its not client proc?